spec/api/v1/admin/patients_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+require 'active_record'
+# rubocop:disable RSpec/MultipleExpectations
```

Comment:
try to avoid disabling of rules and separate expectations into different examples

Reply:
In some cases, I would like to check the fulfillment of several conditions at once. Is there a correct way to implement this?